2.2/Dockerfile.rhel7

```diff
@@ -0,0 +1,44 @@
+FROM openshift/base-rhel7
+
+# This image provides a Ruby 2.0 environment you can use to run your Ruby
+# applications.
+
+MAINTAINER Jakub Hadvig <jhadvig@redhat.com>
+
+EXPOSE 8080
+
+ENV RUBY_VERSION 2.2
+
+LABEL io.k8s.description="Platform for building and running Ruby 2.2 applications" \
+      io.k8s.display-name="Ruby 2.2" \
+      io.openshift.expose-services="8080:http" \
+      io.openshift.tags="builder,ruby,ruby22"
+
+LABEL k8s.io/description="Platform for building and running Ruby 2.2 applications" \
+      k8s.io/display-name="Ruby 2.2" \
+      openshift.io/expose-services="8080:http" \
+      openshift.io/tags="builder,ruby,ruby22"
```

Comment:
It make sense for old images, you're creating a new one, so it's safe to drop it.